GitUI/CommandsDialogs/FormClone.cs

```diff
@@ -401,7 +401,40 @@ private void LoadBranches()
         {
             string from = _NO_TRANSLATE_From.Text;
             Cursor = Cursors.AppStarting;
-            _branchListLoader.LoadAsync(() => Module.GetRemoteServerRefs(from, false, true), UpdateBranches);
+
+            CancellationToken cancellationToken = _branchLoaderSequence.Next();
+            ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
+            {
+                await TaskScheduler.Default;
```

Comment:
❗ `ThreadHelper.JoinableTaskFactory.RunAsync` shall not directly be used!
```suggestion
            ThreadHelper.FileAndForget(async () =>
            {
```